### PR TITLE
Ego boosting leaderboard, reverts player row changes.

### PR DESCRIPTION
**Pull request by stephskardal**

### CODE REVIEW

**Comment by stephskardal** [comment]
Screen Shot 2021-12-04 at 12 58 42 PM
Screen Shot 2021-12-04 at 12 58 48 PM
Screen Shot 2021-12-04 at 12 58 55 PM
Screen Shot 2021-12-04 at 12 59 02 PM

File: leaderboard-ui/src/index.tsx [hunk 1]
```diff
@@ -23,8 +24,20 @@ class App extends React.Component<{}, State> {
 
   async componentDidMount() {
     const players = await get_data();
-    this.setState({players: players});
-    console.log(JSON.stringify(players, null, 2));
+    const name = window.location.search.replace(/^\?I_am=/g, "").replace(/\s/, "").toLowerCase();
+    if(name != '') {
+      var cheerleading = [": You are the best!", ": The only competition is yourself!", ": Ego booster time!!",
```
Comment (mtoothman-upstart) [inline leaderboard-ui/src/index.tsx]:
```
var cheerleading = [
  ": You are the best!", 
  ": The only competition is yourself!", 
  ": Ego booster time!!",
  ": Screenshot and share this!",
  ": You're doing a really good job!",
  ": Dance like there's nobody watching",
  ": Be yourself—everyone else is already taken",
  ": Be the change you wish to see in the world",
  ": Live, laugh, love", 
  ": It's not the number of breaths we take, but the number of moments that take our breath away",
  ": Shoot for the moon. Even if you miss, you'll land among the stars",
  ": If life gives you lemons, make lemonade",
  ": It takes more muscles to frown than it does to smile",
  ": You miss 100% of the shots you don't take"
];
```

Comment (stephskardal) [reply leaderboard-ui/src/index.tsx]: Please submit a PR for this and I'll happily merge. :)

Comment (stephskardal) [reply leaderboard-ui/src/index.tsx]: > 

Nevermind I'll do it now!

**Review by ccarse — APPROVED** [review]

**Pull request merged**